### PR TITLE
Update to elm 0.18

### PR DESCRIPTION
**Pull request by codyplungis**
Jenkins fails to compile the tests because it does not have elm 0.18 installed.

### CODE REVIEW

File: tests/Tests.elm [hunk 1]
```diff
@@ -1,6 +1,7 @@
 module Main exposing (..)
 
-import ElmTest exposing (..)
+import Test exposing (..)
+import Test.Runner.Html
```
Comment (codyplungis) [inline tests/Tests.elm]: For now I went with the html test runner. We might consider the [node test runner](https://github.com/rtfeldman/node-test-runner), but then you have to use `npm install` and I'd rather not have to do that.

**Comment by codyplungis** [comment]
retest this please

**Comment by oakesja** [comment]
retest this please

**Comment by oakesja** [comment]
retest this please

**Comment by oakesja** [comment]
retest this please

**Comment by oakesja** [comment]
retest this please

**Comment by oakesja** [comment]
retest this please

**Pull request merged**